package.json

```diff
@@ -4,31 +4,17 @@
   "version": "0.0.1",
   "repository": "https://github.com/prettier/prettier-chrome-extension.git",
   "license": "MIT",
-  "lint-staged": {
-    "*.{js}": [
-      "eslint --fix",
-      "git add"
-    ],
-    "*.{md,css,html,json,yml}": [
-      "prettier --write",
-      "git add"
-    ]
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "lint-staged"
-    }
-  },
   "scripts": {
+    "start": "webpack --mode=development --watch ",
+    "build": "webpack",
     "fix:other": "yarn prettier --write",
     "fix:js": "yarn test:js --fix",
     "fix": "yarn fix:js && yarn fix:other",
     "install": "yarn build",
     "prettier": "prettier \"**/*.{md,css,html,json,yml}\" --write",
     "test:other": "yarn prettier --list-different",
     "test:js": "eslint --ignore-path .gitignore --ignore-path .prettierignore \"**/*.js\"",
-    "test": "yarn test:other && yarn test:js",
-    "build": "webpack"
+    "test": "yarn test:other && yarn test:js"
```

Comment:
maybe sorting by name is the best here..

Reply:
Sure, I don't have a strong opinion either way